
uint16 WHL_SPD_FL    # 0|14@1+ (0.03125,0) [0|511.96875]
uint16 WHL_SPD_FR    # 16|14@1+ (0.03125,0) [0|511.96875]
uint16 WHL_SPD_RL    # 32|14@1+ (0.03125,0) [0|511.96875]
uint16 WHL_SPD_RR    # 48|14@1+ (0.03125,0) [0|511.96875]
